dspace/config/crosswalks/oai/metadataFormats/oai_openaire.xsl

```diff
@@ -0,0 +1,955 @@
+			</xsl:for-each>
+			
+			<!-- DC PUBLISHER -->
+			<xsl:for-each
+				select="doc:metadata/doc:element[@name='dc']/doc:element[@name='publisher']/doc:element/doc:field[@name='value']">
+				<dc:publisher>
+					<xsl:value-of select="." />
+				</dc:publisher>
+			</xsl:for-each>
+
+			<!-- DC DESCRIPTION ABSTRACT -->
+			<xsl:for-each
+				select="doc:metadata/doc:element[@name='dc']/doc:element[@name='description']/doc:element[@name='abstract']/doc:element/doc:field[@name='value']">
+				<dc:description>
+					<xsl:value-of select="." />
+				</dc:description>
+			</xsl:for-each>
+			
+			<!-- DC SUBJECT -->
+			<xsl:apply-templates
+				select="doc:metadata/doc:element[@name='dc']/doc:element[@name='subject']"
+				mode="datacite">
+			</xsl:apply-templates>
+			
+			<!-- DC DESCRIPTION -->
+			<xsl:for-each
+				select="doc:metadata/doc:element[@name='dc']/doc:element[@name='description']/doc:element[@name='abstract']/doc:field[@name='value']">
+				<dc:description>
+					<xsl:value-of select="." />
+				</dc:description>
+			</xsl:for-each>
+			
+			<!-- DATACITE SIZE -->
+			<xsl:apply-templates
+				select="doc:metadata/doc:element[@name='bundles']/doc:element[@name='bundle']"
+				mode="datacite">
+			</xsl:apply-templates>
+			
+			<xsl:for-each
+				select="doc:metadata/doc:element[@name='dc']/doc:element[@name='relation']/doc:element[@name='ispartofseries']/doc:element/doc:field[@name='value']">
+				<xsl:choose>
+					<xsl:when test="contains(.,';')">
+						<!-- CITATION TITLE -->
+						<oaire:citationTitle>
+							<xsl:value-of select="substring-before(.,';')"/>
+						</oaire:citationTitle>
+						<!-- CITATION VOLUME -->
+						<oaire:citationVolume>
+							<xsl:value-of select="substring-after(.,';')" />
+						</oaire:citationVolume>
+					</xsl:when>
+					<xsl:when test=".!=''">
+						<!-- CITATION TITLE -->
+						<oaire:citationTitle>
+							<xsl:value-of select="."/>
+						</oaire:citationTitle>
+					</xsl:when>
+					<xsl:otherwise>
+					</xsl:otherwise>
+				</xsl:choose>
+			</xsl:for-each>
+			
+			<!-- CITATION ISSUE -->
+			<xsl:for-each
+				select="doc:metadata/doc:element[@name='dc']/doc:element[@name='publisher']/doc:element/doc:element/doc:field[@name='value']">
+				<oaire:citationIssue>
+					<xsl:value-of select="." />
+				</oaire:citationIssue>
+			</xsl:for-each>
+					
+			<xsl:variable name="rightsURI">
+				<xsl:call-template name="resolveRightsURI">
+					<xsl:with-param name="field"
+							select="doc:metadata/doc:element[@name='others']/doc:field[@name='drm']" />
+				</xsl:call-template>
+			</xsl:variable>
+
+			<xsl:for-each
+				select="doc:metadata/doc:element[@name='bundles']/doc:element[@name='bundle']">
+				<xsl:if test="doc:field[@name='name']/text() = 'ORIGINAL'">
+					<xsl:for-each
+						select="doc:element[@name='bitstreams']/doc:element">
+						<oaire:file>
+							<xsl:attribute name="accessRightsURI">
+								<xsl:value-of select="doc:field[@name='drm']/text()"/>
```

Comment:
For a embargoed bitstream it also displays the date: <oaire:file accessRightsURI="embargoed access|||2020-07-22" mimeType="application/pdf" />

Reply:
Ok, noted.